openvpn/build/build.py

```diff
@@ -674,12 +720,25 @@ def post_messages():
 IMAGE_DIR = os.path.abspath(options.image_dir)
 VERBOSE = options.verbose
 PREFIX = options.prefix
+BUILD_TARGET_OS = options.target
 
 if not options.no_clean:
     if os.path.exists(BUILD_DIR):
         shutil.rmtree(BUILD_DIR, False, rmtree_force)
+if BUILD_TARGET_OS == 'Windows' and platform.system() == 'Windows':
+    # windows cannot really build openvpn, this is done on linux with
+    # mingw.
+    options.skip_openvpn = True
+    options.skip_images = True
+if BUILD_TARGET_OS == 'Windows' and platform.system() == 'Linux':
+    # openssl was build on windows and copied back to this host.
+    options.skip_openssl = True
+
 mkdir(BUILD_DIR)
 mkdir(IMAGE_DIR)
+mkdir(STAGE_DIR)
```

Comment:
mh, adding a / to the end seems like a hack, it would be better in I opinion to create the directories prior to the copy (L398/427) as it feels closer to a 'make install'